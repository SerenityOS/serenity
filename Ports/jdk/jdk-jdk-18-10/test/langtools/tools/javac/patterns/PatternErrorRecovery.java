/*
 * @test /nodynamiccopyright/
 * @bug 8268320
 * @summary Verify user-friendly errors are reported for ill-formed pattern.
* @compile/fail/ref=PatternErrorRecovery.out -XDrawDiagnostics -XDshould-stop.at=FLOW --enable-preview -source ${jdk.version} PatternErrorRecovery.java
 * @compile/fail/ref=PatternErrorRecovery-no-preview.out -XDrawDiagnostics -XDshould-stop.at=FLOW PatternErrorRecovery.java
 */
public class PatternErrorRecovery {
    void errorRecoveryNoPattern1(Object o) {
        switch (o) {
            case String: break;
            case Object obj: break;
        }
    }
}
