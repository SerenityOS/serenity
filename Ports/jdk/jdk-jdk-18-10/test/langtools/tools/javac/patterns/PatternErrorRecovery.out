PatternErrorRecovery.java:11:18: compiler.err.pattern.expected
- compiler.note.preview.filename: PatternErrorRecovery.java, DEFAULT
- compiler.note.preview.recompile
1 error
